control/grpc.py

```diff
@@ -145,8 +143,7 @@ def create_subsystem(self, request, context=None):
     def delete_subsystem(self, request, context=None):
         """Deletes a subsystem."""
 
-        self.logger.info(
-            f"Received request to delete {request.subsystem_nqn}")
+        self.logger.info(f"Received request to delete {request.subsystem_nqn}")
```

Comment:
These changes appear to be unrelated -- it's generally better to separate pure formatting stuff.  But, since it's being touched anyway, "delete" -> "delete subsystem":
```suggestion
        self.logger.info(
            f"Received request to delete subsystem {request.subsystem_nqn}")
```

Reply:
In the future, I'll separate out formatting.